### PR TITLE
tests run without torch

### LINKED ISSUE #419: ENH removing torch from the tests
Most of the "data" to compare with the MATLAB versions has been saved via torch. One should aim to remove this line, for instance

https://github.com/kymatio/kymatio/blob/kymatio-dev-frontend/kymatio/scattering2d/tests/test_tensorflow_scattering2d.py#L5

which will affect

https://github.com/kymatio/kymatio/blob/kymatio-dev-frontend/kymatio/scattering2d/tests/test_tensorflow_scattering2d.py#L32

### PR DESCRIPTION
**Pull request by edouardoyallon**
Close #419

### CODE REVIEW

**Comment by edouardoyallon** [comment]
Can we merge?

File: kymatio/scattering2d/tests/test_scattering2d.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 """ This script will test the submodules used by the scattering module"""
 
 import os
+import io
 import numpy as np
 import torch
 import pytest
 from kymatio.scattering2d import Scattering2D
-from kymatio.scattering2d import backend
```
Comment (janden) [inline kymatio/scattering2d/tests/test_scattering2d.py]: Why was this here? Leftover from before?

Comment (edouardoyallon) [reply kymatio/scattering2d/tests/test_scattering2d.py]: yes, I realized one can remove it!

**Comment by edouardoyallon** [comment]
@janden finished!

File: kymatio/scattering2d/tests/test_scattering2d.py (hunk 1 above)
Comment (janden) [reply kymatio/scattering2d/tests/test_scattering2d.py]: Ok. Will move it into a separate commit.
